CONTRIBUTING.md

```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
 
+Install Xcode and Homebrew Mac. Once that is done: 
```

Comment:
I would add a very short explanation of what these are and why they're needed, and hyperlink to their respective landing pages.